script.js

```diff
@@ -0,0 +1,200 @@
+            "summaryTwo": "Utilizing internal software to manage guest book, staff scheduling, POS sales, inventory and product ordering allowed us to control revenue stream."
+        },
+}],
+    "education": [{
+        "institution": "Juno College of Technology",
+        "area": "Web Development",
+        "studyType": "Certificate",
+        "date": "Winter 2020",
+        "courses": [
+            "Web Development Fundamentals",
+            "JavaScript Fundamentals",
+            "Web Development Immersive"
+        ]
+    },
+{
+    "institution": "Seneca College",
+        "area": "Esthetics and Spa Management",
+        "studyType": "Diploma",
+        "date": "Fall 2014 - Spring 2017",
+        "courses": [
+            "Customer Service",
+            "Business ManagementT"
+        ]
+}],
+    "skills": [{
+        "name": "Web Development",
+        "level": "Master",
+        "keywords": [
+        "HTML",
+        "CSS",
+        "Javascript / jQuery",
+        "React",
+        "Working with APIs",
+        "Firebase",
+        "Web Accessibility",
+        "Git / GitHub",
+        "Email Development",
+        "Wireframing",
+        "PSD Conversion",
+        "Responsive Design",
+        "Sketch / Figma",
+        "Adobe Photoshop",
+        "Adobe Illustrator"
+        ]
+    }],
+    "languages": [{
+        "language": "English",
+        "fluency": "Native speaker"
+    }]
+}
+
+let name = resume.basics.name
+let profession = resume.basics.label
+let website = resume.basics.website
+let phone = resume.basics.phone
+let email = resume.basics.email
+let summary = resume.basics.summary
+let resumeSkills = resume.skills[0].keywords
+let skillsToPrint = ""
+let resumeEducation = resume.education
+let educationToPrint = ""
+let resumeExperience = resume.work
+let experienceToPrint = ""
+
+let header = `
+    <div class="wrapper">
+        <h1>Hi! I'm <span>${name}</span></h1>
+        <h2>I'm a ${profession}</h2>
+        <div class="contact">
+            <h3><a href="${website}">www.juliadufresne.com</a></h3>
+            <h3>${email}</h3>
+        </div>
+    </div>
+`
+
+let profile = `
+    <div class="context">
+        <p>${summary}</p>
+    </div>
+`
+
+resumeSkills.forEach((i, index)=>{
+    console.log(i, index)
+    skillsToPrint += `<p>${i}</p>`
+})
+
+let skills = `
+<div class="context">
+    ${skillsToPrint}
+</div>
+`
+
+resumeEducation.forEach((i)=>{
+    let school = i.institution
+    let area = i.area
+    let date = i.date
+
+    educationToPrint += `
+    <div class="context">
+        <div>
+            <p class="school">${school}</p>
+            <p class="area">${area}</p>
+        </div>
+        <p class="date">${date}</p>
+    </div>
+    `
+})
+
+let education = `
+        ${educationToPrint}
+`
+
+resumeExperience.forEach((i)=>{
+    let title = i.position
+    let company = i.company
+    let startDate = i.startDate
+    let endDate = i.endDate
+    let summaryOne = i.summary.summaryOne
```

Comment:
Going back to the summary comment above, you could then render this as:
`
let expSummary = i.summary.map((el)=>{
return <p class="summary"> ${summary[el]}</p>
`